.github/workflows/go.yml

```diff
@@ -37,5 +37,4 @@ jobs:
 
       - name: Test and lint
         run: |
-          export PATH=$PATH:~/go/bin
```

Comment:
I meant to `export PATH=$PATH:./bin`

Reply:
Pls see [ebe4ce8](https://github.com/crypto-power/cryptopower/pull/241/commits/ebe4ce8598bc173d54fcae8042d133a3d141f442)